### DISCUSSION TITLE
不稳定，申请退款，申请退款！！！

### DISCUSSION THREAD

**Discussion by vincentye9682** [getlantern/lantern-forum-cn#1065] [General]
mail: yizunquan@163.com

Payment time: 2024-11-07 11:20:28

Payment method: CMB Credit Card(7419)

游戏充值订单号: P01202411071119396916555

Full name of payee: 广州饶铭网络科技有限公司

Order No.: 2024110722001476771458238537

Merchant order No.: 2024110711194145426

**Comment by MaxMKMax** [comment]
@vincentye9682 您好. 请您提供我支付单据. 我帮您提交上去退款.

**Reply by vincentye9682** [reply]
这是支付截图
![Weixin Image_20241203090126](https://github.com/user-attachments/assets/a79c6447-e78b-48c3-9b67-d7b9f7b5528f)

**Reply by vincentye9682** [reply]
@Mike777Khoo 这个可以了吗？

**Comment by MaxMKMax** [comment]
@vincentye9682 

您好. 我们也想了解不稳定的具体原因.
您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by vincentye9682** [reply]
连接状态经常是时断时续的，我已经卸载掉了

**Reply by MaxMKMax** [reply]
@vincentye9682 好的. 明白. 谢谢!
